Readme.md

```diff
@@ -9,15 +9,13 @@ import (
 	"fmt"
 	"log"
 	"net/http"
-	"os"
 
 	"github.com/apex/gateway"
-	"github.com/aws/aws-lambda-go"
 )
 
 func main() {
 	http.HandleFunc("/", hello)
-	log.Fatal(gateway.ListenAndServe(":3000", nil))
+	log.Fatal(gateway.ListenAndServe("n/a", nil))
```

Comment:
Docs should also be more clear that the address is just there to let you swap with `http.ListenAndServe` but doesn't do anything.

Reply:
sounds reasonable, but now it won't run locally. Maybe we should just have a comment before the ListenAndServe or something

Reply:
In any real code, you need to have to something like 

```go
	listener := http.ListenAndServe
	if isLambda {
		listener = gateway.ListenAndServe
	}
	log.Fatal(listener(port, handler))
```

anyway, so I don't think it matters if the README code technically can't be run locally, as long as it compiles.